packages/server/graphql/mutations/helpers/removeStagesFromMeetings.ts

```diff
@@ -14,12 +14,15 @@ const removeStagesFromMeetings = async (
   const now = new Date()
   const r = await getRethink()
   const activeMeetings = await dataLoader.get('activeMeetingsByTeamId').load(teamId)
+  const completedMeetings = await dataLoader.get('completedMeetingsByTeamId').load(teamId)
+  const meetings = activeMeetings.concat(completedMeetings)
+
   await Promise.all(
-    activeMeetings.map((meeting) => {
+    meetings.map((meeting) => {
       const {id: meetingId, phases} = meeting
       phases.forEach((phase) => {
         // do this inside the loop since it's mutative
-        const {facilitatorStageId} = meeting
+        let {facilitatorStageId} = meeting
```

Comment:
let's get rid of this line & instead on L31, L35 we just pass in `meeting.facilitatorStageId`. 1 less variable to worry about 🎉